src/store/offer/selectors.js

```diff
@@ -0,0 +1,6 @@
+import {NameSpace} from '../root-reducer';
+
+export const getCurrentOffer = (state) => state[NameSpace.OFFER].offer;
+export const getReviews = (state) => state[NameSpace.OFFER].reviews;
+export const getNearbyOffers = (state) => state[NameSpace.OFFER].nearby;
+export const getNotFoundStatus = (state) => state[NameSpace.OFFER].offerNotFound;
```

Comment:
как то очень специфично выходит, может создать один селектор который будет композироваться?
```suggestion
export const getOfferData = (state) => (prop) => state[NameSpace.OFFER][prop];
```